.cloudbuild/graal-build.yaml

```diff
@@ -1,7 +1,14 @@
 steps:
-  - name: oracle/graalvm-ce:20.2.0-java11
+  - name: oracle/graalvm-ce:20.1.0-java11
```

Comment:
Downgrading?

Reply:
Ah yeah, I was getting this error using version 20.2.0:

```
Step #0: Error: Already registered: java.util.zip.ZipFile$CleanableResource.get(ZipFile, File, int)
Step #0: com.oracle.svm.core.util.UserError$UserException: Already registered: java.util.zip.ZipFile$CleanableResource.get(ZipFile, File, int)
Step #0: 	at com.oracle.svm.core.util.UserError.abort(UserError.java:68)
Step #0: 	at com.oracle.svm.core.util.UserError.guarantee(UserError.java:92)
Step #0: 	at com.oracle.svm.hosted.substitute.AnnotationSubstitutionProcessor.register(AnnotationSubstitutionProcessor.java:724)
```

I read on some Slack threads that this is an issue with 20.2.0; not sure how to resolve it just yet but the workaround solution they said was just to use `20.1.0` for now. I'll upgrade to `20.2.0` in a separate PR.